data/data/install.openshift.io_installconfigs.yaml

```diff
@@ -1478,8 +1478,7 @@ spec:
                 type: array
               networkType:
                 description: NetworkType is the type of network to install. The default
```

Comment:
Need to add `default: OVNKubernetes` here if we will add kubebuilder validation.

Reply:
thank you. updated.